src/app/static/src/app/components/plots/table/TableDisplay.vue

```diff
@@ -29,7 +29,7 @@ const defaultColDef = {
     // suppressMenu hides the filter menu which showed on the column title
```

Comment:
also just spotted this 
```suggestion
    // suppressHeaderMenuButton hides the filter menu which showed on the column title
```